src/audio/mixin_mixout.c

```diff
@@ -704,10 +704,15 @@ static int mixin_copy(struct comp_dev *dev)
 		mixed_data_info = mixed_data_info_acquire(mixout_data->mixed_data_info);
 		src_info = find_mixout_source_info(mixed_data_info, dev);
 		if (!src_info) {
-			comp_err(dev, "No source info");
+			/*
```

Comment:
We have a race of PRs here :)
Same issue was fixed just few days ago as part of https://github.com/thesofproject/sof/pull/6179
Here line 695 fixes the same problem:
if (comp_get_state(dev, mixout) != COMP_STATE_ACTIVE)
            continue;

Reply:
@lyakh does #6179 fix your issue ?